src/main/java/com/dev/cinema/Main.java

```diff
@@ -55,13 +59,11 @@ public static void main(String[] args) {
         } catch (AuthenticationException e) {
             System.out.println(e);
         }
-        User user2 = new User("frostpv@gmail.com", "123");
-        try {
-            System.out.println(authenticationService.register(user2.getEmail(),
-                    user2.getPassword()));
-        } catch (AuthenticationException e) {
-            System.out.println(e);
-        }
-
+        System.out.println(user);
+        user = userService.findByEmail("frostpv@gmail.com").get();
+        shoppingCartService.addSession(movieSession, user);
+        System.out.println(shoppingCartService.getByUser(user));
+        shoppingCartService.clear(shoppingCartService.getByUser(user));
+        System.out.println(shoppingCartService.getByUser(user));
```

Comment:
let's maybe save shopping cart in a variable?

Reply:
fixed